### DISCUSSION TITLE
Confused on post-DELETE storage sizes (megabyte, gigabyte)

### DISCUSSION THREAD

**Discussion by givihuda** [dbeaver/dbeaver#17299] [Q&A]
OK so I ran a `DELETE FROM some_bloated_unnecessary_cache_table` on my local DB copy, there's no records now in SELECT, but DBeaver is reporting sizes of hundreds of megabytes on various things with the table even though pre-DELETE the table size was ~1GB.

![image](https://user-images.githubusercontent.com/74785256/183480551-75ebfb32-6ac7-4c6e-968f-75c55fde25fd.png)

Notice while pre-DELETE the "size" (from the first image) was 1.1GB, now it says 960M (megabyte), and when I go into Properties, it says "Data free 1.1GB", "Data Length is 600M", and "Index length 360MB".

![image](https://user-images.githubusercontent.com/74785256/183480825-9b7bb529-84b9-4d9c-a81c-b2658ab21abf.png)

I'm confused by these outputs.

Can someone explain why a recordless table still shows 960MB in DBeaver's table name size section, and why there's still hundreds of megabytes still showing in other places for the table? I don't understand even what all of these numbers mean.

**Comment by E1izabeth** [comment]
Have you updated Tables folder in navigator?

**Reply by givihuda** [reply]
I don't know what this means.

**Reply by E1izabeth** [reply]
Sometimes DBeaver needs to reload metadata after some changes. You can ask DBeaver to do it by clicking on connection or schema in navigator and pressing F5.

**Reply by givihuda** [reply]
I right-clicked the database connection name (localhost -> Databases -> database_schema_name) then selected Refresh, and the number didn't change. The table still says 960M (1G before I ran DELETE all).

Right-clicking the table and going into Properties -> Statistics, it says:

Row Count 0
Avg Row Length 0
Data Length 600M
Max data length 0
Data free 1.1G
Index length 360M
Row format Dynamic

So... why's there such discrepancy in the numbers here after refresh?

**Comment by doraemonxxx** [comment]
This issue still persist. any way to reflect the `Disk Space` ?
